### DISCUSSION TITLE
DatePicker输出值如何改为number类型

### DISCUSSION THREAD

**Discussion by xmsz** [alibaba/formily#3735] [Q&A 2.x]
```
<Field
        name='schedule.date'
        decorator={[FormItem]}
        component={[DatePicker2]}
      />
```
默认是onChange输出是string，但是后端要的是number的类型，这种情况我要如何处理比较好？

**Comment by xiaosi-ss** [comment]
![image](https://user-images.githubusercontent.com/7309546/220511063-bc9a45ad-54af-4ab8-ac90-3b81630b0f67.png)
试一下？  理论是没啥问题的

**Reply by yanhao98** [reply]
我感觉，type 和实际存的值应该没有关系。实际存的值还是是桥接组件的change的值。

**Comment by yanhao98** [comment]
①自己重新桥接一下组件。
②再提交之前把数据再处理一下。
